packages/umpire/package.py

```diff
@@ -115,6 +116,7 @@ class Umpire(CachedCMakePackage, CudaPackage, ROCmPackage):
     depends_on("camp", when="@5.0.0:")
     depends_on("camp@0.2.2:0.2.3", when="@6.0.0")
     depends_on("camp@0.1.0", when="@5.0.0:5.0.1")
+    depends_on("camp@2023.06", when="@2023.06")
```

Comment:
```suggestion
    depends_on("camp@2023.06.0", when="@2023.06.0")
```

Reply:
It matches any patch version if you don't specify it.